perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -173,6 +173,28 @@ public Group call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Create union of two groups, where "operandGroup" is technically set as subgroup of "resultGroup".
```

Comment:
This RPC javadoc is copy/pasted, please update it to represent moveGroup() logic.